ci-operator/jobs/openstack-k8s-operators/neutron-operator/openstack-k8s-operators-neutron-operator-master-presubmits.yaml

```diff
@@ -345,7 +345,7 @@ presubmits:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )neutron-operator-build-deploy,?($|\s.*)
   - agent: kubernetes
-    always_run: true
+    always_run: false
```

Comment:
not sure what was the decision made earlier, but I think that we only have the kuttl jobs as always running (when we have kuttle jobs)

Reply:
@viroel thanks for correcting the confusion. I have kept just kuttl jobs container true and others false.